src/commands/variables/create.ts

```diff
@@ -38,7 +38,19 @@ export default class CreateVariable extends CreateCommand {
         }
         flags._feature = feature
 
-        const params = await this.populateParametersWithZod(CreateVariableDto, this.prompts, flags)
+        let params = await this.populateParametersWithZod(CreateVariableDto, this.prompts, flags)
+        if (!headless) {
+            const { attachToFeature } = await inquirer.prompt([{
+                name: 'attachToFeature',
+                message: 'Attach to a feature?',
```

Comment:
is this the clearest way to phrase this? Should we use similar language to the dashboard like "associate"?

maybe explain what attaching a variable to a feature will do for you in case you didn't know?

like

`Would you like to associate this variable to a feature? Doing so will allow you to serve different values of the variable to different users"

Reply:
good point, I think associate works better

Reply:
although giving a more detailed explanation would be weird to do it just for this one spot, and might also just be too wordy